Makefile

```diff
@@ -58,7 +58,7 @@ test:
 	pytest --junitxml=test-results/junit.xml
 
 test-all:
-	tox
+	tox -- tests/ --ignore=env/ --ignore=venv/ --junitxml=test-results/junit.xml -s
```

Comment:
Moved the `addopts` pytest config from tox.ini to Makefile as postargs
since having addopts in tox.ini can cause a conflict with pytest and
running individual tests. The previous config was only used as part of
the CI/CD pipeline and the new change will not affect the output or
functionality of our pipeline.